src/Emulator.cpp

```diff
@@ -298,6 +298,15 @@ bool romLoaded(Logger* logger, System system, const std::string& path, void* rom
 
   case System::kNintendo:
     ok = romLoadedNes(rom, size);
+
+    if (!ok)
+    {
+      // Fall back to the default strategy, assuming FDS file
+      RA_OnLoadNewRom((BYTE*)rom, size);
+      ok = true;
+    }
+
+    free(rom);
```

Comment:
this results in a double free. after returning from `romLoaded`, `data` is free'd again.

Reply:
Fixed.